### DISCUSSION TITLE
ChordHive - an alternative Ultimate Guitar frontend for bands

### DISCUSSION THREAD

**Discussion by oskarkraemer** [pockethost/pockethost#419] [Show and tell]
Hey,
I built an app using PocketBase that organizes and displays chord and tab sheets for songs in a modern, user-friendly way.
You can even **add notes and sync them effortlessly with your bandmates!**

[chordhive Web](https://chordhive.com)

P.S. Huge thanks to Ben for creating this amazing service—it’s helped my massively to bootstrap my second project now!

![brave_screenshot_chordhive com (3)](https://github.com/user-attachments/assets/ac971bd7-4831-45a8-9aee-41ce4be89de2)
![brave_screenshot_chordhive com (2)](https://github.com/user-attachments/assets/26f1056d-de60-4a59-8c09-08005e11eca6)

**Comment by benallfree** [comment]
Thanks for sharing! Please add to https://github.com/pockethost/pockethost/discussions/419
